remotes/docker/resolver.go

```diff
@@ -284,7 +284,7 @@ func (r *dockerResolver) Resolve(ctx context.Context, ref string) (string, ocisp
 			resp, err := req.doWithRetries(ctx, nil)
 			if err != nil {
 				if errors.Cause(err) == ErrInvalidAuthorization {
-					err = errors.Wrapf(err, "pull access denied, repository does not exist or may require authorization")
+					log.G(ctx).Info("pull access denied, repository does not exist or may require authorization, trying next host")
```

Comment:
Should we attach the original error?

```suggestion
					log.G(ctx).WithError(err).Info("pull access denied, repository does not exist or may require authorization, trying next host")
```

Reply:
I agree.